donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends "pages/base_page.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Ways to Give" %}{% endblock %}
+
+{% block template_name %}app--ways-to-give{% endblock %}
+
+{% block header %}
+<div class="app__header">
+    <h1>{% trans "Ways to Give" %}</h1>
+    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "donate now" %}</a></div>
+</div>
+{% endblock %}
+
+{% block navigation %}{% endblock %}
+
+{% block content %}
+<div class="container">
+    <h2>{% trans "Credit / Debit and PayPal" %}</h2>
+    <p>
+        {% blocktrans trimmed %}
+        <a href="{{ways_to_give_link}}">Go to this link</a> to make a secure online donation right now.
+        {% endblocktrans %}
+    </p>
+
+    <h2>{% trans "Bank Transfer / SEPA" %}</h2>
+    <p><strong>{% trans "New!" %}</strong> {% trans "We are now able to accept donations in Euros through Single European Payment Area (SEPA) transfer, and British Pounds through BACS transfer." %}</p>
+    <p>{% trans "Please remember that donations are deductible for U.S. income tax purposes only." %}</p>
+    <p>
+        {% blocktrans with address="331 E. Evelyn Ave, Mountain View, CA 94041 USA" trimmed %}
+        If your bank requests our physical address, it is {{address}}.
+        {% endblocktrans %}
+    </p>
+
+    <h3>{% trans "Euros:" %}</h3>
+    <p>
+        <b>Account Currency:</b> Euro (EUR)<br>
```

Comment:
Do meta types like `Account Currency:`, `Receiving Bank (Pay to):` need to be localizeable?

Reply:
I don't see why not, but I'll leave that to @TheoChevalier

Reply:
Yes please :) Some of the values need to be localized too (currency names, city names, etc.) You can see here what should be hardcoded (basically bank names + account numbers) https://github.com/mozilla/donate.mozilla.org/blob/master/src/pages/ways-to-give.js